core/config/project_settings.cpp

```diff
@@ -1114,8 +1114,8 @@ ProjectSettings::ProjectSettings() {
 
 	_add_builtin_input_map();
 
-	// Keep the enum values in sync with the `DisplayServer::ScreenOrientation` enum.
 	custom_prop_info["display/window/handheld/orientation"] = PropertyInfo(Variant::INT, "display/window/handheld/orientation", PROPERTY_HINT_ENUM, "Landscape,Portrait,Reverse Landscape,Reverse Portrait,Sensor Landscape,Sensor Portrait,Sensor");
+	custom_prop_info["display/window/vsync/vsync_mode"] = PropertyInfo(Variant::STRING, "display/window/vsync/vsync_mode", PROPERTY_HINT_ENUM, "disabled,on,adaptive,mailbox/triple buffering");
```

Comment:
Capitalize every value for consistency with other property hints, and rename "On" to "Enabled" to put it in contrast with "Disabled":

```suggestion
	custom_prop_info["display/window/vsync/vsync_mode"] = PropertyInfo(Variant::STRING, "display/window/vsync/vsync_mode", PROPERTY_HINT_ENUM, "Disabled,Enabled,Adaptive,Mailbox (Triple Buffering)");
```

Reply:
Ok, changed and replaced "Mailbox(Tripple Buffering)" with just "Mailbox" like @lawnjelly suggested as true triple buffering in Vulkan is only achieved with three images in the swapchain, although it has a similar effect. (I forgot to rename that property earlier)

Reply:
Is it fine like this?

Reply:
Looks good to me now.